### ISSUE TITLE
LABEL elements should be attached to the controls they label. (SortSite issue)

### ISSUE THREAD

**Issue by tpanpalia** [OfficeDev/office-ui-fabric-react#7445]
I am using Office UI Fabric Dropdown in ReactJs. I am testing the application for accessibility issues using SortSite.

It is showing this issue : 
```
LABEL elements should be attached to the controls they label.

Add a FOR attribute set to the ID of the associated control, or wrap the LABEL round the associated control, or use a different element if this isn't a control label. 
```
`label `property is passed to the `Dropdown `component. 
Also, at another instance `Label `component is used with the `Dropdown `component (`for `attribute of `Label `component is same as `id `of `Dropdown`).

**Comment by ecraig12345** [comment]
Hi @tpanpalia, would you mind sharing the URL of the page where you saw this issue, or [creating a codepen](http://aka.ms/fabricpen) to demonstrate?

And just to confirm, is [this](https://www.powermapper.com/products/sortsite/) what you mean by SortSite?

**Comment by cliffkoh** [comment]
Hi @tpanpalia, can you create a codepen repro using the Fabric components the way you're using in your site where the sortsite issue repros? 

https://codepen.io/pen?template=NvBvWx

We are unable to login to your website and a codepen repro will help us isolate the problem. You can alternatively see if the same SortSite errors occur at https://fabricweb.azurewebsites.net/oufr/latest and report a repro using an existing component example.

**Comment by micahgodbolt** [comment]
your error message states `Add a FOR attribute set to the ID of the associated control`

That is exactly what we're doing in our example. for="Dropdown1" and id="Dropdown1". Can you please explain what needs to change?

![image](https://user-images.githubusercontent.com/1434956/50531259-d9c15900-0abb-11e9-87d7-9ad68de7cf88.png)

**Comment by tpanpalia** [comment]
Hi @micahgodbolt ,

This is exactly what we're also doing.
![image](https://user-images.githubusercontent.com/45614905/50580533-a7c24500-0e75-11e9-829a-e7ad46cfd016.png)

But still we are facing the above issue.

**Comment by micahgodbolt** [comment]
yeah, it seems like everything on our end is fine. I'd recommend getting in touch with the a11y testing vendor to see what is required in this case. As of now, there isn't anything we intend to change regarding dropdown labels.

**Comment by tpanpalia** [comment]
Hi @micahgodbolt ,

Thanks for your reply. Please direct us to a11y testing vendors who can help us with the above issue.

**Comment by ecraig12345** [comment]
I think what Micah meant was that you should contact the makers of SortSite to better understand what you should do in this case to get their accessibility checker to pass.
